server/routes.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const db = require('../database/controllers');
+
+const router = express.Router();
+
+const queryAll = (req, res) => {
+  const { song } = req.params;
+  let requestEnd = req.originalUrl.split('/');
+  requestEnd = requestEnd[requestEnd.length - 1];
+  const dbQuery = (requestEnd === 'comments') ? db.getComments : db.getSong;
```

Comment:
interesting method 
however, the requestEnd variable name might be unclear i would suggest to rename it to be clearer i.e.request or requestValue(?) etc

Reply:
I talked to Toby and I am going to split these two back up for clarity